etc/fish_completion/kw.fish

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env fish
+
+# Test if the kw command has subcommands
+function __fish_kw_no_commands
```

Comment:
Why did you add "__" instead of "_"? Also, why did you not used `_kw_autocomplete` to maintain the pattern?

Reply:
This code was based on this (this was made for docker):
https://github.com/docker/cli/blob/master/contrib/completion/fish/docker.fish. I tried to follow the same conventions used in it.

About the name of the file, it must be kw.fish and be placed on one of the directories listed on `fish_complete_path`, otherwise fish won't recognize as the script containing the completions for `kw`. In the source file it can be different, of course, but the user (or the setup script) will need to rename it.

Reply:
If use "__" is a fish shell constraints, that's ok. However, if it is a pattern used by docker I prefer not to use it.

About the path, I prefer to move it to `src` dir because it is a code feature. However, I'm open to new ideas :)

Reply:
About "__": yeah, I found that it is used by these scripts packed with fish: https://github.com/fish-shell/fish-shell/tree/master/share/completions, as you can see, for example, at the completion scripts for apt-get, npm, git and valgrind (all of them use subcommands, just like kw).